components/sidebar/sidebar_channel/sidebar_direct_channel/sidebar_direct_channel.tsx

```diff
@@ -92,10 +92,19 @@ export default class SidebarDirectChannel extends React.PureComponent<Props, Sta
             return icon;
         }
 
+        let className = '';
```

Comment:
Shouldn't this be part of the StatusIconNew itself? Are we planning on using it somewhere where it wouldn't be using these class names?

Reply:
It's because our `status-x` classes belong to the sidebar, and I'm not sure what the reusability of the `StatusIconNew` component will be. I know we're moving a lot of our icons over to the new Mattermost designed ones, so I figured I would keep it as vanilla as possible and let other components dictate the behaviour, at least for now.